test/unit/snowflake_test.js

```diff
@@ -259,6 +259,21 @@ describe('snowflake.createConnection() synchronous errors', function ()
             proxyPort: 'proxyPort'
           },
         errorCode: ErrorCodes.ERR_CONN_CREATE_INVALID_PROXY_PORT
+      },
+      {
+        name: 'invalid row mode',
```

Comment:
do we have such test also on statement level?

Reply:
added